ui/app/helpers/mountable-secret-engines.js

```diff
@@ -134,16 +134,16 @@ const MOUNTABLE_SECRET_ENGINES = [
   },
 ];
 
-// A list of Workflow Identity Federation engines.
-export const WIF_ENGINES = ['aws', 'azure', 'gcp'];
+// A list of Workload Identity Federation engines.
+// Will eventually include Azure and GCP.
+export const WIF_ENGINES = ['aws'];
```

Comment:
were these removed because addition will happen in a follow-on PR?

Reply:
Correct! I use this const later in determining the layout of the configuration details flow, and we're not ready to display GCP or Azure yet. An oversight on my part adding these to begin with.